Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 gemspec
 
-gem 'solidus', github: 'solidusio/solidus', branch: 'v1.1'
```

Comment:
And since this only affects local gem development and not the Gem that users will use in their apps, this is not a very high priority. Or did you encountered any problems, you need to solve now?